tribute-page/css/style.css

```diff
@@ -0,0 +1,94 @@
+html {
```

Comment:
You should configure the linter first for your project.

Reply:
- [ ] Kindly make sure you rephrase your comments and make them in a friendly and polite way.
- [ ] Also, always explain why the changes are suggested